apps/files_external/3rdparty/icewind/streams/src/Wrapper.php

```diff
@@ -92,7 +92,11 @@ public function stream_eof() {
 	}
 
 	public function stream_close() {
-		return fclose($this->source);
+		if (is_resource($this->source)
```

Comment:
I tried this on my installation, however I got a syntax error. Is it possible that you forgot one closing `)`?

```suggestion
		if (is_resource($this->source))
```

Reply:
That's embarrasing lol

I was coding it directly on my instance and re-typed it on GitHub's UI itself for a quick pull request. I didn't catch the closing `)`. I've added your suggested commit, thanks so much!